scripts/time-filter-fast.js

```diff
@@ -0,0 +1,116 @@
+import {
+  textSummary
+} from 'https://jslib.k6.io/k6-summary/0.0.3/index.js'
+import http from 'k6/http'
+import {
+  check,
+  sleep
+} from 'k6'
+import {
+  generateRandomCoordinate,
+  destinations,
+  timeFilterScenarios as scenarios,
+  setThresholdsForScenarios,
+  countries,
+  summaryTrendStats,
+  deleteTimeFilterMetrics,
+  reportPerDestination
+} from './common.js'
+
+export const options = {
```

Comment:
I think you can move options to a common file

Reply:
done